static/src/javascripts/bootstraps/dotcom-rendering-commercial.js

```diff
@@ -37,7 +37,11 @@ const commercialModules: Array<Array<any>> = [
     ['cm-lotame-data-extract', initLotameDataExtract],
 ];
 
-if (false && !commercialFeatures.adFree) {
+if (
```

Comment:
Do we want to use 'control' as the default here rather than false? It feels a bit unclear as the types don't seem to match.

Reply:
Good idea. I thought of using a string as default, to match the types, but didn't think deep enough.

Reply:
https://github.com/guardian/frontend/pull/21789